services/tasks/local.go

```diff
@@ -89,10 +87,6 @@ func init() {
 
 func initFunc(ic *plugin.InitContext) (interface{}, error) {
 	config := ic.Config.(*Config)
-	runtimes, err := loadV1Runtimes(ic)
```

Comment:
Perhaps I should file a issue to track a propose that how to add a go plugin for task instead of external shim binary. With the containerd plugin design, the developers can build customized containerd with imported plugins, like https://github.com/fuweid/embedshim which imports runtime task management for runC-like container.

I would like to build lightweight containerd to manage runC-like container without shim and the user can use a control binary to create container, just like what bash shell does right now. The plugin task runtime management is the key for the goal.

Reply:
> Perhaps I should file a issue to track a propose that how to add a go plugin for task instead of external shim binary. With the containerd plugin design, the developers can build customized containerd with imported plugins, like https://github.com/fuweid/embedshim which imports runtime task management for runC-like container.

it would be useful to support loading runtimes that the user has registered with the plugin
In the future we can keep using user-defined plugins with matching names and task manager as the default runtime